src/components/MediaLink/Audio.vue

```diff
@@ -79,20 +80,26 @@
               <span>{{ getTimeFromSeconds(this.currentTime) }}</span>
               <span>{{ getTimeFromSeconds(this.totalTime) }}</span>
             </div>
+
+          </div>
+
+          <div> 
+            <button class="blip-change-Playback-Rate" @click="changePlaybackRate">x{{ this.playbackRate }}</button>
```

Comment:
The visualization of this button is this 
![image](https://user-images.githubusercontent.com/3072069/163871711-66b0b916-fe4c-4248-bf7f-451587c69ecf.png)

Does it right?

Reply:
It seems like it wasn't used the blip-ds css pattern or buttons patterns no cards, right? DId a Product Designer approve it?

Reply:
Blip-ds was not used because when applying it, it did not conform to what was designed in the prototype at https://www.figma.com/file/Zmv9EDTBEIVo3YLazvN025/%5BDESK%5D%5BWEB%5D%5BHANDOFF%5D-Desk -Routines?node-id=670%3A53258

<img width="503" alt="image" src="https://user-images.githubusercontent.com/101811613/163997597-4b2942ac-fec6-4dee-9a9d-fb168fb6d650.png">


**It was validated with a Product Designer and it was approved to be using "button"** 

Using blip-ds it is not possible to change the color of the button to be like the message balloon
<img width="495" alt="image" src="https://user-images.githubusercontent.com/101811613/163997236-561c0e1d-0e7f-4d2f-9a14-208e8339e88b.png">

The visualization of this button should be this: 
<img width="452" alt="image" src="https://user-images.githubusercontent.com/101811613/163998086-66dac265-78a9-4aa5-91c8-2cd1aeb4f2ea.png">